packages/cli/__tests__/zosfiles/__integration__/view/ds/__snapshots__/cli.files.view.ds.integration.test.ts.snap

```diff
@@ -52,9 +52,9 @@ exports[`View data set should display the help 1`] = `
 
    --range | -R (string)
 
-      The range of records to return in either of SSS-EEE where SSS is the start and
-      EEE is the end of records or SSS,NNN where is is the start and NNN is the number
-      of records retrieved.
+      The range of records to return in either \`SSS-EEE\` or \`SSS,NNN\` format. SSS
+      identifies the start record, EEE identifies the end record to be retrieved, and
+      NNN identifies the number of records to be retrieved.
```

Comment:
suggestion: NNN identifies the number of **retrieved** records.

Reply:
Thank you Ana!!